shared/hedvig-ui/json-schema-form.tsx

```diff
@@ -180,9 +180,9 @@ const formatInitialFormData = (
 ): Record<string, unknown> => {
   const properties = schema.properties!
   const formData = { ...initialFormData }
-  Object.entries(properties).forEach((property) => {
-    const [key, value] = property as [string, JSONSchema7]
-    formData[key] = initialFormData[key] ?? value?.default ?? undefined
+  Object.entries(properties).forEach(([key, value]) => {
+    formData[key] =
+      initialFormData[key] ?? (value as JSONSchema7).default ?? undefined
```

Comment:
```suggestion
      initialFormData[key] ?? (value as JSONSchema7)?.default
```

(or something like that?)

Always a bit skeptical using `undefined` explicitly 🧐

Reply:
Aparently the JSON schema form prefers undefined to null 🤦